### ISSUE TITLE
Error: 'JavaInfo' value has no field or method 'transitive_deps'

### ISSUE THREAD

**Issue by sgowroji** [bazelbuild/rules_jvm_external#946]
https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/3271#018a4433-bbcd-42f9-a119-99623fce8f89

Platform : Multiple

Logs : 
```
Error: 'JavaInfo' value has no field or method 'transitive_deps'
```
Culprit : https://github.com/bazelbuild/bazel/commit/72058be0628e84741232cb116a7199eb92638f84

CC Greenteam @mai93

**Comment by sgowroji** [comment]
Dupe https://github.com/bazelbuild/rules_jvm_external/issues/676

**Comment by sgowroji** [comment]
Hi @shs96c, As mentioned here https://github.com/bazelbuild/rules_jvm_external/pull/947#issuecomment-1727930564. Did you get a chance the above issue. Thanks!

**Comment by sgowroji** [comment]
Hi @[jin](https://github.com/jin) [& @shs96c](https://github.com/shs96c), Can you take a look of the above issue. Thanks!

**Comment by cheister** [comment]
I believe this was fixed with https://github.com/bazelbuild/rules_jvm_external/pull/999

**Comment by JohnLKramer** [comment]
Sorry but I'm seeing this problem in bazel 7.0.2. Does that include this fix?

**Comment by mai93** [comment]
I think it was fixed by upgrading `rules_kotlin` to 1.9 in https://github.com/bazelbuild/rules_jvm_external/pull/999 not by a change in bazel
